canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -329,10 +354,13 @@ class FlexibleTable extends React.Component {
 		if (!this.props.data[displayedRowIndex].disabled) {
 			if (overSelectOption) { // Checkbox is clicked
 				let current = this.props.selectedRows ? this.props.selectedRows : [];
-				if (checked) {
+				if (data.selectMultipleRows) { // multiple rows selected/deselected using shift key
+					current = this.handleCheckedMultipleRows(data.lastCheckedRow, displayedRowIndex, checked); // displayedRowIndex or dataRowIndex???
```

Comment:
Use displayedRowIndex or dataRowIndex???

Reply:
What is the difference?

Reply:
`displayedRowIndex` is the index. And `dataRowIndex` is the originalRowIndex. Same comment as above. I will keep the current code of using `displayedRowIndex`  for now.